scarb/src/ops/cachepath.rs

```diff
@@ -0,0 +1,12 @@
+use anyhow::Result;
+
+use crate::core::Config;
+
+#[tracing::instrument(skip_all, level = "debug")]
+pub fn cache_path(config: &Config) -> Result<()> {
+    let path = config.dirs().cache_dir.path_unchecked();
+    if path.exists() {
```

Comment:
I think we should output the path even if it is not existing in the filesystem yet.